sources/packages/api/src/route-controllers/utils/custom-validators/sin-validator.ts

```diff
@@ -0,0 +1,58 @@
+import {
+  ValidatorConstraintInterface,
+  ValidatorConstraint,
+  registerDecorator,
+  ValidationOptions,
+  ValidationArguments,
+} from "class-validator";
+import { SIN_MAX_LENGTH } from "../../../database/entities";
+
+/**
+ * SIN validation algorithm (Luhn algorithm).
+ * ! This logic is also present on form.io client side validations.
+ */
+@ValidatorConstraint()
+export class IsValidSINConstraint implements ValidatorConstraintInterface {
```

Comment:
Just wondering if this class needs to be exported since it is only used during registration of the decorator which is in the same file.

Reply:
Good catch, no it is not needed. I just removed it.